satellite/marketing/marketingweb/server.go

```diff
@@ -0,0 +1,119 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package marketingweb
+
+import (
+	"context"
+	"html/template"
+	"net"
+	"net/http"
+	"runtime"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+)
+
+// Error is satellite marketing error type
+var (
+	Error       = errs.Class("satellite marketing error")
+	_, fp, _, _ = runtime.Caller(0)
```

Comment:
May be we should move this code to `storj-sim` and use config values for static locations? Because this may not work with binaries, it will return path where the file was during build. On the other hand `with storj-sim` we can more or less assume that it is built from source code and static files are in place

Reply:
This is the code already in `storj-sim/network.go` which is similar to what I have in the server file for setting fp. Also I don't believe it returns the path on build but on invocation as stated in the godoc : https://golang.org/pkg/runtime/#Caller correct me if I'm missing something please.

		`// TODO: find source file, to set static path
		_, filename, _, ok := runtime.Caller(0)
		if !ok {
			return nil, errs.Combine(processes.Close(), errs.New("no caller information"))
		}
		storjRoot := strings.TrimSuffix(filename, "/cmd/storj-sim/network.go")`

Reply:
Yeah, we shouldn't definitely use it in a global, for prod, it should come from a flag. For storj-sim, it might be fine.

Reply:
@rikysya I just made sure passing 0 to the runtime.Caller() func works the way I expected by creating a test file with only that function and printing the return value by building and executing the binary. 

No matter where you run the binary it will print the absolute path to the file invoking the function. If you duplicate the above process you will see that it's not returning the path on build as you suggested but on invocation like the go-doc describes.

@egonelbre I'll add a flag to storj-sim for prod as you suggested.

Reply:
@rikysya @egonelbre I pushed the requested changes.